api-service/controller/routing.go

```diff
@@ -45,6 +45,7 @@ func SetupRoutesForAPIController(router *mux.Router, ctrl APIControllerInterface
 	router.HandleFunc("/environments", ctrl.ListEnvironments).Methods("GET")
 	router.HandleFunc("/clusters", ctrl.SearchClusters).Methods("GET")
 	router.HandleFunc("/databases", ctrl.SearchDatabases).Methods("GET")
+	router.HandleFunc("/databases/defaulttags", ctrl.GetDefaultDatabasesTags).Methods("GET")
```

Comment:
Are you sure that "/databases/defaulttags" is the best name?
I'm thinking, in the future will be added a endpoint like /databases/{dbname} though aa db can only identified with hostname+dbname?
Can you add a dash between default and tags?
Can a endpoint like /setting/database-dafault-tag-choiches be a better idea?

Reply:
`/settings/database-dafault-tag-choiches` is surely a better idea!
I'm going to create a new controller and service for settings then!

Reply:
Ok